test/functional/mining_basic.py

```diff
@@ -28,12 +28,16 @@
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import (
     assert_equal,
+    assert_greater_than_or_equal,
     assert_raises_rpc_error,
     get_fee,
 )
 from test_framework.wallet import MiniWallet
 
 
+DIFFICULTY_ADJUSTMENT_INTERVAL = 144
+MAX_FUTURE_BLOCK_TIME = 2 * 3600
+MAX_TIMEWARP = 600
```

Comment:
In e929054e12210353812f440c685a23329e7040f7 "Add timewarp attack mitigation test"

This constant is unused in this commit.

Reply:
I'll move it to the other commit, or find a way to use it here, if I need to retouch.